### ISSUE TITLE
Named form actions don't URI decode

### ISSUE THREAD

**Issue by willfarrell** [sveltejs/kit#9057]
### Describe the bug

I've been testing out our new app using the latest version of sveltekit on AWS using CloudFront (w/ origin request policies) and Lambda Function URLs. When using named form actions I found that CloudFront will return a 400 because the `/` is Invalid. After encoding to `%2F` SvelteKit doesn't recognize it.

I've reached out to AWS already, they suggested the encoding mentioned above. I've followed up asking for documentation, will report back if they have any. I can submit a PR to `sveltekit-adapter-aws` to re-encode the leading `/` as a workaround, but doesn't feel like the right solution (cc @MikeBild).

Are other CDN provider running into this? Should decoding be handled by SvelteKit or the adaptor? Should there be a option the changes the prefix to something else that doesn't need URI encoding? Should a notice be added to the docs about AWS CloudFront?

### Reproduction

Blocked by AWS CloudFront, but works locally
```
<form method="POST" action="?/anything">
...
</form>
```

AWS suggestion, Sveltekit doesn't recognize, falls back to `default`
```
<form method="POST" action="?%2Fanything">
...
</form>
```

Currently live example: https://qa.datastream.org/en-CA/login?/username

### System Info

```Shell
Unable to run within AWS Lambda (arm64, nodejs18.x, esm)
```

### Severity

serious, but I can work around it

**Comment by Conduitry** [comment]
Something like this has come up before in #7098, but I have not read everything carefully.

**Comment by willfarrell** [comment]
Thanks, I'll give it a read.

**Comment by willfarrell** [comment]
I'm going to close, as it is a duplicate of https://github.com/sveltejs/kit/issues/7098. I'll comment there any new feedback there that I get from AWS.
